README.md

```diff
@@ -3,20 +3,21 @@
 
 ### przegląd historii zmian zachodzących w plikach  
 ### branching - rozgałęzianie kodu
-### git init - zainizjalizowanie repozytorium    
-### git clean - usuwanie plików (nie da się przywrócić)
-### git reset - odwrotność git add
-### git checkout - usuwanie zmian do poprzedniego stanu
-### git rm - usuwanie z repozytorium (można przywrócić)
-### git log - wyświetla historie pliku
-### git revert - odwrócenie zmian  z poprzedniego commitu i zapisanie go jako nowy commit
-### git merge - łączenie branchy
-### git status - status kodu
-### git stash - stos używany przy nadpisywaniu branchy
-### git stash pop - przepisanie treści stosu
-### git push - zmiany wysyłane są z lokalnego na zdalne
-### git fetch - pobranie zmian
-### git rebase - łaczenie ze sobą commitów
+### `git init` - zainizjalizowanie repozytorium    
+### `git clean` - usuwanie plików (nie da się przywrócić)
+### `git reset` - odwrotność git add
+### `git checkout` - usuwanie zmian do poprzedniego stanu
+### `git rm` - usuwanie z repozytorium (można przywrócić)
+### `git log` - wyświetla historie pliku
+### `git revert` - odwrócenie zmian  z poprzedniego commitu i zapisanie go jako nowy commit
+### `git merge` - łączenie branchy
+### `git status` - status kodu
+### `git stash` - stos używany przy nadpisywaniu branchy
+### `git stash pop` - przepisanie treści stosu
+### `git push` - zmiany wysyłane są z lokalnego na zdalne
+### `git fetch` - pobranie zmian
+### `git rebase` - łaczenie ze sobą commitów
+### `git checkout -b "nazwa"` - tworzenie nowej gałęzi i automatyczne przejście do niej
```

Comment:
zgrupuj polecenia git checkout razem i dodaj `checkout -` - opisem co to robi :)

Reply:
dodane 👍